spec/def_initialize/accessors_builder_spec.rb

```diff
@@ -0,0 +1,44 @@
+RSpec.describe DefInitialize::AccessorsBuilder do
+  subject { described_class.build(accessors, options) }
+
+  let(:accessors) { [':x', ':y', ':z'] }
+
+  context 'only readers' do
+    let(:options) { { readers_mode: :public, writers_mode: nil } }
+
+    specify do
+      is_expected.to include('public')
+      is_expected.to include('attr_reader :x, :y, :z')
```

Comment:
The order of expectation does not guarantee the order of statements `public` and `attr_reader :x, :y, :z`

Reply:
Ok, I will rewrite to check equality